documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -169,9 +169,12 @@ Finally, the operator will provide API response to the API invoker (Step 14).
 
 **Technical ruleset for the Frontend flow**
 
-> If all API usecases point to the need of On-net scneario and where the consumption device and authentication device are the same, the front end flow must be used. eg. NumberVerification
+The technical ruleset is applicable only after a subproject has agreed to use a 3-legged authentication flow. This ruleset provides a recommendation which will help API providers to align on the 3-legged flow and help with aggregation.
+
+If all API usecases point to the need of On-net scneario and where the consumption device and authentication device are the same, the front end flow should be used. eg. NumberVerification
```

Comment:
Small typo:
```suggestion
If all API usecases point to the need of On-net scenario and where the consumption device and authentication device are the same, the front end flow should be used. eg. NumberVerification
```

Reply:
@shilpa-padgaonkar I think you should apply this commit as well to correct the typos detected by @rartych